pandas/tests/series/test_constructors.py

```diff
@@ -2107,6 +2107,14 @@ def test_series_string_inference_scalar(self):
             ser = Series("a", index=[1])
         tm.assert_series_equal(ser, expected)
 
+    def test_series_string_inference_array_string_dtype(self):
+        # GH#54496
+        pa = pytest.importorskip("pyarrow")
+        expected = Series(["a", "b"], dtype=pd.ArrowDtype(pa.string()))
+        with pd.option_context("future.infer_string", True):
+            ser = Series(np.array(["a", "b"]))
```

Comment:
It might be good to specify dtype explictly here.

Not sure how numpy's string dtype work is coming along, but just in case that becomes the default, it would be good to future-proof this.

Reply:
I'd prefer to keep this, we should infer the default type as arrow backed strings for now, that's the mos common use case for users